KillProof Module.csproj

```diff
@@ -77,8 +77,7 @@
       <HintPath>packages\AsyncClipboardService.1.7.1\lib\net45\AsyncClipboardService.dll</HintPath>
     </Reference>
     <Reference Include="Blish HUD">
-      <HintPath>..\Blish-HUD\Blish HUD\bin\x64\Debug\Blish HUD.exe</HintPath>
-      <Private>False</Private>
```

Comment:
You need to double check that the Blish HUD exe is not set to copy local if this is not included.  Without this and if copy local is not set correctly, your module will package a copy of the entire Blish HUD exe within it which will make your module size huge.

Reply:
https://github.com/blish-hud/KillProof-Module/pull/9/commits/5e7d4aed75cf40b2872c13f541203d361a94b854